src/main/java/org/broadinstitute/hellbender/tools/exome/Case2PoNTargetMapper.java

```diff
@@ -96,7 +96,6 @@ public RealMatrix fromPoNToCaseCounts(final RealMatrix ponCounts) {
      * @return never {@code null} a read-count collection with the row order according to the case read count target order.
      */
     public ReadCountCollection fromPoNtoCaseCountCollection(final RealMatrix counts, final List<String> countColumnNames) {
-        return new ReadCountCollection(SetUniqueList.setUniqueList(new ArrayList<>(outputTargets)),SetUniqueList.setUniqueList(new ArrayList<>(countColumnNames)),
-                fromPoNToCaseCounts(counts));
+        return new ReadCountCollection(outputTargets, countColumnNames, fromPoNToCaseCounts(counts));
```

Comment:
The previous version was creating a copy of the lists.  Are you sure that is okay to move to reference?